<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
        <meta name="description" content="" />
        <meta name="author" content="" />
        <title>Gaël Fortin</title>
        <link rel="icon" type="image/x-icon" href="assets/img/favicon.ico" />
        <!-- Font Awesome icons (free version)-->
        <script src="https://use.fontawesome.com/releases/v5.15.1/js/all.js" crossorigin="anonymous"></script>
        <!-- Google fonts-->
        <link href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:500,700" rel="stylesheet" type="text/css" />
        <link href="https://fonts.googleapis.com/css?family=Muli:400,400i,800,800i" rel="stylesheet" type="text/css" />
        <!-- Core theme CSS (includes Bootstrap)-->
        <link href="css/styles.css" rel="stylesheet" />
    </head>
    <body id="page-top">
        <!-- Navigation-->
        <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
            <a class="navbar-brand js-scroll-trigger" href="#page-top">
                <span class="d-block d-lg-none">Gaël Fortin</span>
                <span class="d-none d-lg-block"><img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="assets/img/profile.jpg" alt="" /></span>
            </a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
            <div class="collapse navbar-collapse" id="navbarSupportedContent">
                <ul class="navbar-nav">
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#about">About</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#experience">Experience</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#education">Education</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#skills">Skills</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#interests">Interests</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#Publications">Publications</a></li>
                </ul>
            </div>
        </nav>
        <!-- Page Content-->
        <div class="container-fluid p-0">
            <!-- About-->
            <section class="resume-section" id="about">
                <div class="resume-section-content">
                    <h1 class="mb-0">
                        Gaël
                        <span class="text-primary">Fortin</span>
                    </h1>
                    <div class="subheading mb-5">
                        Research scientist, oncology and bioinformatics
                    </div>
                    <p class="lead mb-5">My research interests involve applying interdisciplinary approaches to answer biological questions. I am 
                    especially interested in the use of bioinformatics for the analysis of high-throughput experiments to understand oncogenesis and drug resistance. My research is focused 
                	on the identification of mutation combinations that drive chemoresistance in acute myeloid leukemia. I develop open-source R packages and apps 
                	to make bioinformatics accessible for all.</p>
                    <div class="subheading mb-5">
                        <a href="mailto:fortin.gael@outlook.fr">fortin.gael@outlook.fr</a>
                        <p class="lead mb-5"> (+33) 6 28 25 93 33</p>
                    </div>
                    <div class="social-icons">
                        <a class="social-icon" href="https://www.linkedin.com/in/ga%C3%ABl-fortinnouel/"><i class="fab fa-linkedin-in"></i></a>
                        <a class="social-icon" href="https://github.com/gaelfortin/"><i class="fab fa-github"></i></a>
                        <a class="social-icon" href="https://twitter.com/FortinGael"><i class="fab fa-twitter"></i></a>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Experience-->
            <section class="resume-section" id="experience">
                <div class="resume-section-content">
                    <h2 class="mb-5">Experience</h2>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">PhD student</h3>
                            <div class="subheading mb-3">Puissant Lab, Institut de Recherche Saint-Louis</div>
                            <p>
                            	<b>Phosphogo: a user-friendly app to predict kinases activity and relevance with phosphoproteomics.</b></p>
                            <p>In the last years, significant improvments in the generation and the analysis of phosphoproteomic data have allowed scientists to 
                            interrogate pathway and cell activity directly, without using other intermediates such as RNA or protein levels. However, phosphoproteomics
                        	still lack comparative and predictive tools accessible for all. Current available predictive tools that indicate kinase-substrate relationships 
                        	require knowledge in bioinformatics. Phosphogo, a user-friendly app built on R and shiny, allows scientists to interrogate their phosphoproteomics datasets
                        	without prior bioinformatics skills and to predict kinase activity and relevance in the cellular context. More on <a class="social-icon" href="https://github.com/gaelfortin/phosphogo"><i class="fab fa-github"></i></a></p>
                        	<p>
                        		<b>Deciphering mutationally-induced chemoresistance in acute myeloid leukemia.</b></p>
                    		<p>Acute myeloid leukemia is the most common acute leukemia diagnosed in adults. Despite an increase in the understanding of the disease, the 5-year overall survival rate of patients remains low (around 10%) mainly due to common chemoresistant relapses after remission. Specific combinations of mutations have been identified as potential drivers of chemoresistance. This project aims to validate these combinations and understand the underlying mechanisms leading to chemoresistance in mouse models of leukemia. This project is supervised by <a href="mailto:alexandre.puissant@inserm.fr">Dr Alexandre Puissant</a> and <a href="mailto:nina.fenouille@inserm.fr">Dr Nina Fenouille</a>.</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">October 2020 - Present</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Master 2 intern</h3>
                            <div class="subheading mb-3">Puissant Lab, Institut de Recherche Saint-Louis</div>
                            <p>
                            	<b>An integrated (phospho-)proteomic approach to decipher chemoresistance in acute myeloid leukemia.</b></p>
                            <p>Acute myeloid leukemia is the most common adult leukemia diagnosed in Europe. Its poor survival rate is mainly due to the rise of a chemoresistant disease. I used an integrated omics approach including transcriptomi	cs, proteomics and phosphoproteomics on chemosensitive and chemoresistant leukemia to identify active signaling pathways and nominate cellular vulnerabilities. This project was supervised by <a href="mailto:antoine.forget@ucsf.edu">Dr Antoine Forget</a>.</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">November 2019 - July 2020</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Master 1 intern</h3>
                            <div class="subheading mb-3">Rothstein Lab, Columbia University (New York)</div>
                            <p><b>Deciphering the cisplatin tolerance pathways through genetic interacctions in the yeast <i>Saccharomyces cerevisiae</i></b></p>
                            <p>Cisplatin is a drug commonly used in the treatment of many cancers. However, cancer cells often develop resistance to this compound. Understanding how cisplatin arises is thus a major challenge. I used high-throughput screens on yeast to identify genes and pathways involved in drug resistance. I developped bioinformatic approaches to analyse, visualize and interrogate genetic networks. During this internship I also supervised an intern for 2 months. This project was supervised by <a href="mailto:rr381@cumc.columbia.edu">Dr Robert Reid</a>./p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">April 2019-August 2019</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Licence 3 intern</h3>
                            <div class="subheading mb-3">Rossoll Lab, Mayo Clinic Jacksonville (Florida)</div>
                            <p><b>Study of the disassembly of TDP-43 aggregates by nuclear transport factors in an amyotrophic lateral sclerosis model.</b></p>
                            <p>Amyotrophic lateral scleroris is a neurodegenerative and neuromuscular disease that lead to the death of motoneurons that control voluntary muscles. The presence of TDP-43 aggregates in these cells is a hallmark of ALS. During this internship, I studied how these aggregates can be disassembled by nuclear transport factors. This project was supervised by <a href="mailto:khalil.bilal@mayo.edu"> Dr Bilal Khalil</a>.</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">May 2018-August 2018</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">President</h3>
                            <div class="subheading mb-3">Association Biologie et Génétique Appliquée</div>
                            <p>President of the Association BGA, the student association of the Magistère Européen de Génétique. During this one-year presidency I managed a team of 14 dedicated people in the organization of multiple festive or charity events. I learned team management and team working, communcation, working with strict and short deadlines.</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">March 2018-March 2019</span></div>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Education-->
            <section class="resume-section" id="education">
                <div class="resume-section-content">
                    <h2 class="mb-5">Education</h2>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Practical Deep Learning for Coders</h3>
                            <div>Self-training on machine learning and deep learning with the corresponding course from <a href="https://www.fast.ai/">fast.ai</a>. This course covers machine learning (random forest) and various deep learning domains such as image recognition, natural language processing, tabular data predictions as well as deep learning from the foundations (neural network architecture, stochastic gradient descent...).</div>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">January 2021 - Present</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Magistère Européen de Génétique</h3>
                            <div class="subheading mb-3">Université de Paris</div>
                            <div>A 3-year bachelor and master degree in genetics with specialization in oncogenesis and bioinformatics. </div>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">September 2017 - June 2020</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Mouse Genetics</h3>
                            <div class="subheading mb-3">Institut Pasteur</div>
                            <div>A 5-week intensive training on mouse models. This course, mainly consisting of practical lessons, covers major aspects of mice use in research: genetics, behaviour, neurology, development, as well as stemness (through cell guided differenciation and iPSCs production).</div>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">January 2020 - February 2020</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Licence 1 and 2 of Molecular and Cellular Sciences, international section</h3>
                            <div class="subheading mb-3">Université Rennes 1</div>
                            <div>1st and 2nd year of Licence in an international program.</div>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">September 2015 - June 2017</span></div>
                    </div>
                    
                </div>
            </section>
            <hr class="m-0" />
            <!-- Skills-->
            <section class="resume-section" id="skills">
                <div class="resume-section-content">
                    <h2 class="mb-5">Skills</h2>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Programming Languages & Tools</h3>
                            <div class="subheading mb-3">R - Every day, building packages and GUIs with shiny.</div>
                            <div class="subheading mb-3">LaTeX - Building complex reports.</div>
                            <div class="subheading mb-3">Deep learning - Self-learned, tabular data analysis and predictions.</div>
                            <div class="subheading mb-3">Python - Acquainted.</div>
                            <div class="subheading mb-3">Unix - Daily use of Linux-based distributions, standard use of bash.</div>
                            <div></div>
                        </div>
                    </div>
                    <div>
                    	<p></p>
                    	 	
                    </div>
                    <div class="subheading mb-3">Workflow</div>
                    <ul class="fa-ul mb-0">
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Mobile-First, Responsive Design
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Cross Browser Testing & Debugging
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Cross Functional Teams
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Agile Development & Scrum
                        </li>
                    </ul>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Interests-->
            <section class="resume-section" id="interests">
                <div class="resume-section-content">
                    <h2 class="mb-5">Interests</h2>
                    <p>Apart from being a web developer, I enjoy most of my time being outdoors. In the winter, I am an avid skier and novice ice climber. During the warmer months here in Colorado, I enjoy mountain biking, free climbing, and kayaking.</p>
                    <p class="mb-0">When forced indoors, I follow a number of sci-fi and fantasy genre movies and television shows, I am an aspiring chef, and I spend a large amount of my free time exploring the latest technology advancements in the front-end web development world.
                    HTML
                	Vulgarisation scientifique
                	Deep-learning Self-study
                	</p>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Publications-->
            <section class="resume-section" id="Publications">
                <div class="resume-section-content">
                    <h2 class="mb-5">Publications</h2>
                    <ul class="fa-ul mb-0">
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Google Analytics Certified Developer
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Mobile Web Specialist - Google Certification
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            1
                            <sup>st</sup>
                            Place - University of Colorado Boulder - Emerging Tech Competition 2009
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            1
                            <sup>st</sup>
                            Place - University of Colorado Boulder - Adobe Creative Jam 2008 (UI Design Category)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            2
                            <sup>nd</sup>
                            Place - University of Colorado Boulder - Emerging Tech Competition 2008
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            1
                            <sup>st</sup>
                            Place - James Buchanan High School - Hackathon 2006
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            3
                            <sup>rd</sup>
                            Place - James Buchanan High School - Hackathon 2005
                        </li>
                    </ul>
                </div>
            </section>
        </div>
        <!-- Bootstrap core JS-->
        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
        <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js"></script>
        <!-- Third party plugin JS-->
        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.4.1/jquery.easing.min.js"></script>
        <!-- Core theme JS-->
        <script src="js/scripts.js"></script>
    </body>
</html>
